### PR TITLE
feat: [EBF-4719] expose get last used card

### PR DESCRIPTION
**Pull request by hoanghoang93**
expose function to get last used payment method:
Android:

https://github.com/braintree/braintree-android-drop-in/blob/3d0843ce33c1e87866365778793a93bf5580654b/README.md?plain=1#L110

IOS:

https://github.com/braintree/braintree-ios-drop-in/blob/6bd10f16745d95755dba743a5448b1ee2c5513cb/README.md?plain=1#L170

### CODE REVIEW

File: android/src/main/java/tech/bam/RNBraintreeDropIn/RNBraintreeDropInModule.java [hunk 1]
```diff
@@ -77,6 +82,50 @@ public void show(final ReadableMap options, final Promise promise) {
...
+          @Override
+          public void onError(Exception exception) {
+              // an error occurred
+              promise.reject(GET_LAST_USED_CARD_ERROR, exception.getMessage());
+          }
+
+          @Override
+          public void onResult(DropInResult result) {
+              mPromise = promise;
+              if (result.getPaymentMethodType() != null && result.getPaymentMethodNonce() != null) {
```
Comment (quocthangit247) [inline android/src/main/java/tech/bam/RNBraintreeDropIn/RNBraintreeDropInModule.java:103]: can we export this as variables and then reuse it?
```suggestion
PaymentMethodType  paymentMethodType = result.getPaymentMethodType();
PaymentMethodNonce paymentMethod = result.getPaymentMethodNonce();

              if (paymentMethodType != null && paymentMethod != null) {
```

Comment (quocthangit247) [reply android/src/main/java/tech/bam/RNBraintreeDropIn/RNBraintreeDropInModule.java:103]: cc @dangkhoa2708 @ducng-ehero

Comment (vilipwong-eh) [inline android/src/main/java/tech/bam/RNBraintreeDropIn/RNBraintreeDropInModule.java:103]:
```suggestion
              if (result.getPaymentMethodType() == null && result.getPaymentMethodNonce() == null) {
              // there was no existing payment method
                  promise.reject(GET_LAST_USED_CARD_ERROR, "No existing last used card");
                  return;
              }
```
remove nested if, better readability ?

**Review by vilipwong-eh — APPROVED** [review]

**Pull request merged**